### PR TITLE
Add a heroku runbook

### PR DESCRIPTION
**Pull request by jivdhaliwal**
# What
Documentation on how we use Heroku to deploy the app

# Why
To help anyone wanting to setup the app

# Screenshots

# Notes

### CODE REVIEW

File: docs/runbook/heroku.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Deploying to heroku
+
+The app has been designed to easily deploy to heroku.
+
+## Pipelines
+
+We recommend [creating a pipeline](https://devcenter.heroku.com/articles/pipelines) with a staging and production app.
+
+Each app requires the [heroku/node.js buildpack](https://devcenter.heroku.com/articles/buildpacks) and uses the following addons: [Heroku Postgres](https://elements.heroku.com/addons/heroku-postgresql), [Heroku Scheduler](https://elements.heroku.com/addons/scheduler).
```
Comment (antonyoneill) [inline docs/runbook/heroku.md:9]: Might be worth highlighting that they should probably use a paid for dyno, and they'll need to add a postgress add on

File: docs/runbook/heroku.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Deploying to heroku
+
+The app has been designed to easily deploy to heroku.
```
Comment (JessicaMadeTech) [inline docs/runbook/heroku.md:3]: Capitalisation

File: docs/runbook/heroku.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+After enabling the heroku scheduler addon for each environment in your pipeline (staging, production), create a new daily job and set the command to:
+
+```
+npm run cleandb
+```
+
+## Build timeouts
+
+By default, heroku sets a [60 second build timeout](https://devcenter.heroku.com/articles/limits#boot-timeout). If a build takes longer than 60 seconds to complete, the build will fail and prevent a deploy from taking place.
```
Comment (JessicaMadeTech) [inline docs/runbook/heroku.md:37]: Capitalisation

File: docs/runbook/heroku.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+The majority of builds take less than 60 seconds, but on rare occasions they can take more (this happens when a new package is installed, which increases the build time as we have to wait for the installation to complete).
+
+To get around this, we suggest setting the build timeout time to 90 seconds (which can be done via the [build timeout tool](https://tools.heroku.support/limits/boot_timeout))
```
Comment (JessicaMadeTech) [inline docs/runbook/heroku.md:41]: Full stop at the end of the sentence.

**Pull request merged**